docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
+
+services:
+  db:
+    image: arm64v8/mysql:oracle
+    ports:
+      - 3306:3306
+    environment:
+      - MYSQL_ROOT_PASSWORD=Abcd1234
```

Comment:
Ez a fájl ignorálható.
A `docker-compose` fájl egy Docker-es beállításokat tartalmazó fájl, nekem csak azért kellett, hogy a másik gépemen is fel tudjak gyorsan húzni egy MySQL adatbázist.